Dockerfile

```diff
@@ -45,7 +45,7 @@ COPY . /code/
 # copy built react and node libraries for production mode
 COPY --from=node-prod-deps /usr/src/app/package-lock.json /code/package-lock.json
 COPY --from=node-prod-deps /usr/src/app/webpack-stats.json /code/webpack-stats.json
-COPY --from=node-prod-deps /usr/src/app/assets/images /code/static
+COPY --from=node-prod-deps /usr/src/app/assets/dist/static /code/static
 COPY --from=node-prod-deps /usr/src/app/assets /code/assets
```

Comment:
This fixes this error 
![Screen Shot 2019-06-14 at 10 44 42 AM](https://user-images.githubusercontent.com/8579775/59517370-77a1b800-8e91-11e9-863e-926b9c9030fb.png)

Reply:
This error is never happening now with roboto not importing